turbo.json

```diff
@@ -21,9 +25,21 @@
       "interactive": false,
       "cache": false
     },
+    "build:dev": {
```

Comment:
This allows you to build a production instance of the application locally. If we only use prisma --no-engine then it won’t be able to connect to your local DB because it won’t have the postgres client.